### ISSUE TITLE
Resoning behind skipping customer-data cleanup ?

### ISSUE THREAD

**Issue by maquowicz** [bluepayment-plugin/magento-2.x-plugin#12]
Hi!

view/frontend/web/js/model/place-order.js override skips some important cleanups - Is there any particular reason behind this ? 
This causes some unusual behaviors (using built in m2 gateways - checkmo/banktransfer) like duplicating customer addresses after order placed, customer-data out of sync (front vs back). Have no access to Autopay sandbox and forced to perform live transactions only which makes figuring this out close to impossible.

Referring to following part of core place-order.js@41
```
            .done(
                function (response) {
                    var clearData = {
                        'selectedShippingAddress': null,
                        'shippingAddressFromData': null,
                        'newCustomerShippingAddress': null,
                        'selectedShippingRate': null,
                        'selectedPaymentMethod': null,
                        'selectedBillingAddress': null,
                        'billingAddressFromData': null,
                        'newCustomerBillingAddress': null
                    };

                    if (response.responseType !== 'error') {
                        customerData.set('checkout-data', clearData);
                    }
                }
            )
```

Any help would be appreciated.
Many Thanks.

**Comment by zalazdi** [comment]
@maquowicz 
Thank you for your submission. We are analyzing the changes introduced.

If you have any further questions, please contact us at the following address https://developers.autopay.pl/kontakt

**Comment by zalazdi** [comment]
@maquowicz 
Problem resolved in version v2.22.9 - https://github.com/bluepayment-plugin/magento-2.x-plugin/releases/tag/v2.22.9.

Thank you for report!
